Google.PowerShell/Compute/GceHealthCheckCmdlets.cs

```diff
@@ -0,0 +1,545 @@
+                }
+                catch (Exception e)
+                {
+                    exceptions.Add(e);
+                }
+            }
+
+            if (Https || !Http)
+            {
+                HttpsHealthChecksResource.GetRequest request = Service.HttpsHealthChecks.Get(Project, Name);
+                try
+                {
+                    healthChecks.Add(request.Execute());
+                }
+                catch (Exception e)
+                {
+                    exceptions.Add(e);
+                }
+            }
+
+            if (healthChecks.Count == 0)
+            {
+                if (exceptions.Count == 1)
+                {
+                    throw exceptions[0];
+                }
+                else
+                {
+                    throw new AggregateException(
+                        $"The health check named {Name} in project {Project} can not be found", exceptions);
+                }
+            }
+            else
+            {
+                return healthChecks;
+            }
+        }
+
+        private IEnumerable<object> GetAllProjectHealthChecks()
+        {
+            if (Http || !Https)
+            {
+                HttpHealthChecksResource.ListRequest request = Service.HttpHealthChecks.List(Project);
+                do
+                {
+                    HttpHealthCheckList response = request.Execute();
+                    if (response.Items != null)
+                    {
+                        foreach (HttpHealthCheck healthCheck in response.Items)
+                        {
+                            yield return healthCheck;
+                        }
+                    }
+                    request.PageToken = response.NextPageToken;
+                } while (!Stopping && request.PageToken != null);
+            }
+
+            if (Https || !Http)
+            {
+                HttpsHealthChecksResource.ListRequest request = Service.HttpsHealthChecks.List(Project);
+                do
+                {
+                    HttpsHealthCheckList response = request.Execute();
+                    if (response.Items != null)
+                    {
+                        foreach (HttpsHealthCheck healthCheck in response.Items)
+                        {
+                            yield return healthCheck;
+                        }
+                    }
+                    request.PageToken = response.NextPageToken;
+                } while (!Stopping && request.PageToken != null);
+            }
+        }
+    }
+
+    /// <summary>
+    /// <para type="synopsis">
+    /// Removes a Google Compute Engine health check.
+    /// </para>
+    /// <para type="description">
+    /// Removes a Google Compute Engine Health check. Use this cmdlet to remove both http and https health
+    /// checks.
+    /// </para>
+    /// </summary>
+    [Cmdlet(VerbsCommon.Remove, "GceHealthCheck", SupportsShouldProcess = true)]
```

Comment:
Probably good to add an example here. Remove all HealthChecks matching a specific string and passing `-WhatIf`?